go.mod

```diff
@@ -3,10 +3,8 @@ module github.com/lieut-data/mattermost-plugin-api
 go 1.13
 
 require (
-	github.com/mattermost/mattermost-server/v5 v5.99.99
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20191203094615-d0d6ce0a7014
```

Comment:
To double check: is this `v5.3.2` expected?

Reply:
Yes it is. When using a specific commit (which is needed because there no releases contains these changes), go uses `v5.3.2` as a prefix. Don't ask me why :man_shrugging: